lib/reverseproxy/revers_proxy.go

```diff
@@ -0,0 +1,130 @@
+package reverseproxy
+
+import (
+	"flag"
+	"io"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/bytesutil"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+)
+
+const (
+	targetURL     = "vm-target-url"
+	forwardHeader = "X-Forwarded-For"
+)
+
+var (
+	maxIdleConnsPerBackend = flag.Int("maxIdleConnsPerBackend", 100, "The maximum number of idle connections vmauth can open per each backend host")
+)
+
+var bbPool bytesutil.ByteBufferPool
+
+// ReversProxy represents simple revers proxy based on http.Client
+type ReversProxy struct {
+	client *http.Client
+}
+
+// New initialize revers proxy
+func New() *ReversProxy {
```

Comment:
Could you make naming consistent for this struct and name it `ReverseProxy`?

Reply:
fixed